maven-artifacts/poms/frameworkPom.xml

```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <name>Checker Framework Core Library</name>
+    <groupId>org.checkerframework</groupId>
+    <artifactId>framework</artifactId>
+
+    <!-- The comments in the version number are used to do a text substitution. Don't remove them.
+         See checker-framework/bin/poms/updateVersion.sh -->
+    <version><!-- checker-framework-version -->2.6.0<!-- /checker-framework-version --></version>
+
+    <distributionManagement>
+        <repository>
+            <id>checker-releases</id>
+            <url>https://checkerframework.org/dev/m2-repo</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>dataflow</artifactId>
```

Comment:
This is blocked on having `stubparser` and `annotation-tools` Maven dependencies.